testworld/start_test.go

```diff
@@ -0,0 +1,51 @@
+// This is the starting point for all Testworld tests
+// +build testworld
+
+package testworld
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/centrifuge/go-centrifuge/config"
+)
+
+var isRunningOnCI = len(os.Getenv("TRAVIS")) != 0
+
+// Adjust these based on local testing requirments, please revert for CI server
+var configFile = "configs/local.json"
+var runPOAGeth = !isRunningOnCI
+
+// make this false if you want to make the tests run faster locally, but revert before committing to repo
+var runMigrations = !isRunningOnCI
+
+// doctorFord manages the hosts
+var doctorFord *hostManager
+
+func TestMain(m *testing.M) {
+	c, err := loadConfig(configFile)
+	if err != nil {
+		panic(err)
+	}
+	if runPOAGeth {
+		// NOTE that we don't bring down geth automatically right now because this must only be used for local testing purposes
```

Comment:
can we bring it down at the end of this `TestMain` just in case that has been started by this process? Meaning:
```
alreadyRunning := startPOAGeth()
...
if !alreadyRunning {
   stopPOAGeth()
}
```
That way we do not have lingering process, even testing locally.

Reply:
As I commented, right now it's kind of annoying locally when it does that because then the run+test+debug process for a test takes a long time, instead of adding a lot of flags I decided to keep it simple and stupid. On CI the testworld doesn't start geth by it self. it relies on already running one.